src/components/select-button-group/index.js

```diff
@@ -14,8 +14,9 @@ const SelectButtonGroup = ({
     noBorder,
     extraClass,
 }) => {
-    return values.map(value => (
+    return values.map((value, key) => (
         <SelectButton
+            key={key}
```

Comment:
While this would probably work in our situation (as we don't actually modify the list of values), I think it's supposedly best practice to put a value that won't change here (that is also unique). What I mean by that is if we were to remove a value from the values we use, and insert a new one, then the keys would change (which ends up causing issues).

I'm fine with it if we don't think we'll ever need to modify the values passed in though.